changelog/unreleased/enhancement-add-keyboard-navigation-to-spaces

```diff
@@ -0,0 +1,6 @@
+Enhancement: Add keyboard navigation to spaces
+
+We've added keyboard navigation and actions to spaces.
```

Comment:
In the spaces overview it is now possible to navigate through the individual spaces and perform actions using the keyboard